bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -113,10 +113,9 @@ protected void updateHomeEvent(HomeEvent homeEvent) {
 
     private void addEventIfKnownObject(HomeEvent homeEvent, @Nullable String objectId) {
         if (objectId == null) {
-            return;
+            handler.getActiveChildren(FeatureArea.SECURITY).filter(child -> child.getId().equals(objectId))
+                    .forEach(child -> child.setNewData(homeEvent));
         }
```

Comment:
That looks strange, you are now comparing child id with null.

Reply:
Yes, my mistake

Reply:
Corrected